src/dimensions.js

```diff
@@ -19,6 +19,11 @@ jQuery.each( { Height: "height", Width: "width" }, function( name, type ) {
 
 				if ( jQuery.isWindow( elem ) ) {
 
+					// $( window ).outerWidth/Height return w/h including scrollbars (gh-1729)
+					if ( funcName.indexOf( "outer" ) === 0 ) {
+						return elem[ "inner" + name ];
+					}
+
 					// As of 5/8/2012 this will yield incorrect results for Mobile Safari, but there
```

Comment:
For such use cases emulators should be fine. Also, BrowserStack now has real iOS devices for iOS 7 & 8.

I've just checked and the issue exists in iOS 6 but doesn't in 7/8/9. We still support iOS 6 in 1.12/2.2 but not in 3.0 and this change is breaking so this is remark is moot now.

This makes sense, btw, as iOS 7 experienced a major redesign, the Safari scrolling behavior was also changed in a number of ways so it's not surprising behavior changed here as well.